Gemfile

```diff
@@ -22,7 +22,6 @@ gem "rails-i18n"
 gem "sass-rails", ">= 6"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 4.0"
-gem "kaminari"
```

Comment:
we accidentally wrote two same of coding of this, so we delete one to prevent any confuse.